### PR TITLE
Restart process if a migration has run

### PR DESCRIPTION
**Pull request by stayradiated**
If the database has changed because of a database migration, we now restart the app processes to make sure that Ruby detects the changes.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -111,12 +111,26 @@ const main = async () => {
...
       })
 
-      await heroku.run({
+      const logs = await heroku.run({
         appName: target.mainAppName,
         type: 'worker',
         command: ['bundle', 'exec', 'rake', 'db:migrate'],
       })
 
+      console.log(logs)
```
Comment (ro-savage) [inline src/index.ts:120]: Log

File: src/index.ts [hunk 2]
```diff
@@ -111,12 +111,26 @@ const main = async () => {
...
+
+      if (/migrating/.test(logs)) {
```
Comment (ro-savage) [inline src/index.ts:122]: Will this only appear if there is a migration? Or does this appear every time?

I think we run the migration every time right? But only if there is an actual change does something happen.

File: src/index.ts (hunk 1 above)
Comment (stayradiated) [reply src/index.ts]: Yep, this was me debugging that we could actually get the migration logs from the command.

File: src/index.ts (hunk 2 above)
Comment (stayradiated) [reply src/index.ts]: This only appears when there is a migration to perform.

**Review by ro-savage — APPROVED** [review]
Sweet. Should be good to go.

Since we have to restart the dyno on migrations no matter what. We don't really need to test about the downtime.

**Pull request merged**